entity-framework/core/get-started/uwp/getting-started.md

```diff
@@ -2,137 +2,150 @@
 title: Getting Started on UWP - New Database - EF Core
 author: rowanmiller
 ms.author: divega
-ms.date: 10/27/2016
+ms.date: 08/08/2018
 ms.assetid: a0ae2f21-1eef-43c6-83ad-92275f9c0727
 uid: core/get-started/uwp/getting-started
 ---
 
 # Getting Started with EF Core on Universal Windows Platform (UWP) with a New Database
 
-> [!NOTE]
-> This tutorial uses EF Core 2.0.1 (released alongside ASP.NET Core and .NET Core SDK 2.0.3). EF Core 2.0.0 lacks some crucial bug fixes required for a good UWP experience.
+In this tutorial, you build a Universal Windows Platform (UWP) application that performs basic data access against a local SQLite database using Entity Framework.
 
-In this walkthrough, you will build a Universal Windows Platform (UWP) application that performs basic data access against a local SQLite database using Entity Framework.
-
-> [!IMPORTANT]
-> **Consider avoiding anonymous types in LINQ queries on UWP**. Deploying a UWP application to the app store requires your application to be compiled with .NET Native. Queries with anonymous types have worse performance on .NET Native.
```

Comment:
Reviewing this on my phone so, not sure I get the full picture. This paragraph seems to be effectively removed. I am actually ok with that, but I think we should have a link to a list of UWP limitations, probably a new section on platforms.md.